www-servers/nginx/patches/nginx-1.19.7.patchset

```diff
@@ -1,7 +1,7 @@
-From 0d0bb4ae9552f7a9a2c87a27e910b3ad1a4a92bc Mon Sep 17 00:00:00 2001
-From: Sergei Reznikov <diver@gelios.net>
-Date: Wed, 24 Jul 2019 01:43:25 +0300
-Subject: Haiku fixes
+From 9bb9a4bd5106c08fece6ef89052be877c31cdbb8 Mon Sep 17 00:00:00 2001
+From: Ian Corbitt <ian@ihatemyisp.net>
+Date: Fri, 5 Mar 2021 21:59:00 +0000
+Subject: Haiku fixes based off Sergei Reznikov <diver@gelios.net>
```

Comment:
Would be nice if you could keep the credits from prior committer

Reply:
Ah, so the from can be manually changed. I've changed it.